pkg/cluster/nodes/node.go

```diff
@@ -221,16 +154,6 @@ func (n *Node) FixMounts() error {
 	if err := n.Command("mount", "-o", "remount,ro", "/sys").Run(); err != nil {
```

Comment:
this should probably move into ENTRYPOINT and we can get rid of SIGUSR1, unfortunately there's the userns remap hackery so we need to plumb through info there first.

might do that in this PR so we can have a single breaking image change point

Reply:
I did this